xinference/core/model.py

```diff
@@ -120,42 +129,65 @@ async def _wrap_generator(self, ret: Any):
         else:
             return ret
 
-    async def _call_wrapper(self, _wrapper):
+    async def _call_wrapper(self, _wrapper: Callable):
         if self._lock is None:
             return await asyncio.to_thread(_wrapper)
         else:
             async with self._lock:
                 return await asyncio.to_thread(_wrapper)
 
+    async def _call_async_wrapper(self, _wrapper: Callable):
+        return await asyncio.create_task(_wrapper())
```

Comment:
Currently, the `ModelActor` is a `StatelessActor`, be aware that this method is not guarded by the asyncio Lock.